models/Individual.js

```diff
@@ -0,0 +1,90 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+import { isEmail,isStrongPassword,isURL} from 'validator';
+
+const individualSchema = new Schema({
+    name:{
+        type:String,
+        required:true,
+        trim:true,
+    },
+    email:{
+        type: String,
+        required: true,
+        unique:true,
+        trim:true,
+        validate: [ isEmail, 'Please provide a valid email ID' ]
+    },
+    password:{
```

Comment:
create a hashing function for saving the encrypted password before saving the detail in the schema

Reply:
created